doc/devel/index.rst

```diff
@@ -12,14 +12,18 @@ The Matplotlib Developers' Guide
 .. toctree::
    :maxdepth: 2
 
-   coding_guide.rst
-   portable_code.rst
-   license.rst
```

Comment:
Are you talking about the license.rst? It doesn't really fit in the developers section. There is currently a link to it somewhere in the user documentation.